### ISSUE TITLE
Multiple mister cifs mounts

### ISSUE THREAD

**Issue by rogersta99** [danmons/retronas#188]
I have my original mister cifs mount setup following @danmons video tutorial and it works as it should.   I setup a second mister and followed the same tutorial but when I use the ./cifs_mount.sh command it fails to mount as it seems to be trying to point itself towards the IP address of the original mister I setup.  Is there something I am missing or is this something that is not currently supported?

**Comment by danmons** [comment]
Can you please copy/paste the `cifs_mount.ini` files you have on both MiSTer systems? (Remove the "password" section).

**Comment by rogersta99** [comment]
This is the "first/original" mister setup:
#=========   USER OPTIONS   =========
#You can edit these user options or make an ini file with the same
#name as the script, i.e. mount_cifs.ini, containing the same options.

#Your CIFS Server, i.e. your NAS name or its IP address.
SERVER="retrosmb"

#The share name on the Server.
SHARE="mister"

#The user name, leave blank for guest access.
USERNAME="pi"

#The user password, irrelevant (leave blank) for guest access.
PASSWORD="1"

#Optional user domain, when in doubt leave blank.
DOMAIN=""

#Local directory/directories where the share will be mounted.
#- It can ba a single directory, i.e. "_CIFS", so the remote share, i.e. \\NAS\MiSTer
#  will be directly mounted on /media/fat/_CIFS (/media/fat is the root of the SD card).
#- It can be a pipe "|" separated list of directories, i.e. "Amiga|C64|NES|SNES",
#  so the share subdirectiories with those names,
#  i.e. \\NAS\MiSTer\Amiga, \\NAS\MiSTer\C64, \\NAS\MiSTer\NES and \\NAS\MiSTer\SNES
#  will be mounted on local /media/fat/Amiga, /media/fat/C64, /media/fat/NES and /media/fat/SNES.
#- It can be an asterisk "*": when SINGLE_CIFS_CONNECTION="true",
#  all the directories in the remote share will be listed and mounted locally,
#  except the special ones (i.e. linux and config);
#  when SINGLE_CIFS_CONNECTION="false" all the directories in the SD root,
#  except the special ones (i.e. linux and config), will be mounted when one
#  with a matching name is found on the remote share.
LOCAL_DIR="*"

#Optional additional mount options, when in doubt leave blank.
#If you have problems not related to username/password, you can try "vers=2.0" or "vers=3.0".
#ADDITIONAL_MOUNT_OPTIONS="ver=3.1"

#"true" in order to wait for the CIFS server to be reachable;
#useful when using this script at boot time.
WAIT_FOR_SERVER="true"

#"true" for automounting CIFS shares at boot time;
#it will create start/kill scripts in /etc/network/if-up.d and /etc/network/if-down.d.
MOUNT_AT_BOOT="true"

**Comment by sairuk** [comment]
I don't think we can really speak for the MiSTer scripts not working correctly, we have not control over those. There is nothing on the RetroNAS side that would prevent two devices mounting the share.

I know I read somewhere when I was first having issues setting up cifs mounts that for some MiSTers mounting at boot just doesn't work and it is expected the users will need to run the script manually, this may have been in regards to updated scripts I can't really recall. I've never dug into why. 

I don't have two MiSTers here and my current one does work at boot so I wont be able to debug it here. 

Some ideas of what to do to debug further if you want to eliminate the MiSTer/RetroNAS as an issue
- Swap the sdcard or network locations as a test to see if the behaviour is transferred.
- Turn off the both MiSTers and then boot the second one and see if it connects on boot
- Confirm the cifs_mount scripts are the same version on both MiSTers

**Comment by rogersta99** [comment]
Both mister are connected to the network via Wi-Fi.  I will give that a shot and report back.

**Comment by danmons** [comment]
Looks like all MiSTer devices have the same MAC address, defined in software.

You can't have two devices on the same network with the same MAC address.  There's a provided script with MiSTer to change one of your MiSTers MAC addresses:

https://github.com/MiSTer-devel/Scripts_MiSTer/blob/master/mac_address_change.sh

Give that a try and see if it solves your issue.
